Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/MicrosoftGraphMail.yml

```
@@ -8,112 +11,140 @@ configuration:
   name: private_key
   required: false
   type: 14
+  section: Connect
 - display: Fetch incidents
   name: isFetch
   required: false
   type: 8
+  section: Collect
 - display: Email address from which to fetch incidents (e.g. "example@demisto.com")
   name: mailbox_to_fetch
   required: false
   type: 0
+  section: Collect
 - defaultvalue: Inbox
   display: Name of the folder from which to fetch incidents (supports Folder ID and sub-folders e.g. Inbox/Phishing)
   name: folder_to_fetch
   required: false
   type: 0
+  section: Collect
 - defaultvalue: 15 minutes
   display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days)
   name: first_fetch
   required: false
   type: 0
+  section: Collect
 - additionalinfo: The timeout of the HTTP requests sent to Microsoft Graph API (in seconds).
   defaultvalue: '10'
   display: HTTP Timeout
   name: timeout
   required: false
   type: 0
+  section: Connect
+  advanced: true
 - defaultvalue: '50'
   display: Maximum number of emails to pull per fetch
   name: fetch_limit
   required: false
   type: 0
+  section: Collect
 - display: Trust any certificate (not secure)
   name: insecure
   required: false
   type: 8
+  section: Connect
+  advanced: true
 - display: Use system proxy settings
   name: proxy
   required: false
   type: 8
+  section: Connect
+  advanced: true
 - display: Use a self deployed Azure Application
   name: self_deployed
   required: false
   type: 8
+  section: Connect
+  advanced: true
 - display: Incident type
   name: incidentType
   required: false
   type: 13
+  section: Connect
 - display: ID or Client ID - see Detailed Instructions (?)
   hidden: true
   name: _auth_id
   required: false
   type: 0
+  section: Connect
 - display: Token or Tenant ID - see Detailed Instructions (?)
   hidden: true
   name: _tenant_id
   required: false
   type: 0
+  section: Connect
 - display: Key or Client Secret (Deprecated)
   hidden: true
   name: enc_key
   required: false
   type: 4
+  section: Connect
 - display: ID or Client ID - see Detailed Instructions (?) (Deprecated)
   hidden: true
   name: auth_id
   required: false
   type: 4
+  section: Connect
 - display: Token or Tenant ID - see Detailed Instructions (?) (Deprecated)
   hidden: true
   name: tenant_id
   required: false
   type: 4
+  section: Connect
 - additionalinfo: Whether to fetch incidents with the entire email body, or just an email body preview, mark if the full email body is required.
   defaultvalue: 'false'
   display: Display the entire email body
   name: display_full_email_body
   type: 8
+  section: Collect
+  advanced: true
 - additionalinfo: Use this parameter to determine how long backward to look in the search for incidents that were created before the last run time and did not match the query when they were created.
   defaultvalue: 0
   display: 'Advanced: Time in minutes to look back when fetching emails'
   name: look_back
   required: false
   type: 0
+  section: Collect
```

```suggestion
  section: Collect
  advanced: true
```
Because it is "Advanced" in the display, but maybe it is not wanted in this case